core/start-service.sh

```diff
@@ -31,8 +35,9 @@ ADDITIONAL_KOPANO_PACKAGES=$(echo "$ADDITIONAL_KOPANO_PACKAGES" | tr -d '"')
 	fi
 done
 
-mkdir -p /kopano/data/attachments /kopano/data/kapi-kvs /tmp/"$SERVICE_TO_START" /var/run/kopano /var/lib/kopano-grapi
+mkdir -p /tmp/"$SERVICE_TO_START" /var/run/kopano
 
+# TODO is this still required now that we won't modify configuration mounted to /etc/kopano?
```

Comment:
@RobinvG  with this PR config files are moved to /tmp before being updated. This makes it seem like the below if is not required anymore. What do you think?